WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -0,0 +1,171 @@
+<% provide(:title, @competition.name) %>
+
+<% if @admin_view %>
+  <h2>Announcements</h2>
```

Comment:
I did update the post results link [here](https://github.com/cubing/worldcubeassociation.org/commit/148b5e7a6b358d7ced0a57157c7e000b8e7a8df8#diff-d4c5068a7b12ed233fbb1b00f3e34e0dR155).

There is currently no link to announce the competition results, although the `/competitions/*/post/announcment` route does work. I'll add a link as part of porting competitions_manage.php to rails.